ui/png-icons/src/main/res/drawable/chart_box_outline.xml

```diff
@@ -0,0 +1,8 @@
+<!-- drawable/chart_box_outline.xml -->
```

Comment:
Please remove the comment. It just mentions the file name and does not add useful information